src/styles.css

```diff
@@ -87,9 +87,10 @@
   margin-top: 0.625rem;
```

Comment:
My recent learning has been that margins should probably be in pixels. REM is device dependent (literally the size of an "m"). So margins will be adjusted accordingly and that may not be desired. But will defer to your judgment

Reply:
Last I heard we were an all-rem-all-the-time shop. Also, I didn't add that. This is a larger discussion that we should  have on Slack once we have designer help. Until then I think we should just keep doing what we've been doing.